src/segment.go

```diff
@@ -263,6 +265,7 @@ func (segment *Segment) mapSegmentWithWriter(env environmentInfo) error {
 		Memory:        &memory{},
 		Angular:       &angular{},
 		PHP:           &php{},
+		Processor:     &processor{},
```

Comment:
Do we want to add this here or rename the `memory` one to `device`?

Reply:
It's pure WIP. I have no idea for the name. We could merge the host information(cpu, load, memory) into one segment will all properties available for the template.

Reply:
When using a plain text template that could be possible even without all the styling when they're sequential. To facilitate non-sequential segments, we could also cache the response for one run in environment?

Reply:
of course, if we cache it we could have one segment usable multiple times without the burden of getting data.

Reply:
opted for a sysinfo segment. Used gopsutil mem package. Still WIP.